tests/php/BlockExtendPluginTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Tests for class BlockExtendPlugin.
+ */
+
+namespace XWP\BlockExtendTest;
+
+use WP_Mock;
+use Mockery;
+use XWP\BlockExtend\BlockExtendPlugin;
+use XWP\BlockExtend\Plugin;
+
+/**
+ * Tests for class BlockExtendPlugin.
+ */
+class BlockExtendPluginTest extends BlockExtendTestCase {
+
+	/**
+	 * Test init.
+	 *
+	 * @covers XWP\BlockExtend\BlockExtendPlugin::init()
+	 */
+	public function test_init() {
+		$plugin = new BlockExtendPlugin( Mockery::mock( Plugin::class ) );
```

Comment:
By mocking the `XWP\BlockExtend\Plugin` class we decouple the test from dependencies.

Reply:
😄